app/src/main/res/layout/activity_search_results.xml

```diff
@@ -3,47 +3,53 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="com.mmmoussa.iqra.SearchResultsActivity" >
+    tools:context="com.mmmoussa.iqra.SearchResultsActivity">
 
     <TextView
+        android:id="@+id/arabicVerse"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="Arabic Verse"
-        android:id="@+id/arabicVerse"
         android:layout_alignParentTop="true"
-        android:gravity="right"
-        android:textSize="28sp"
         android:background="@color/colorPrimary"
-        android:textColor="@android:color/white"
+        android:gravity="right"
+        android:paddingBottom="20dp"
         android:paddingLeft="30dp"
         android:paddingRight="30dp"
-        android:paddingBottom="20dp" />
+        android:text="Arabic Verse"
+        android:textColor="@android:color/white"
+        android:textSize="28sp" />
 
     <TextView
+        android:id="@+id/resultCount"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="New Text"
-        android:textColor="#FFFFFF"
-        android:background="#CCCCCC"
-        android:id="@+id/resultCount"
-        android:layout_below="@+id/arabicVerse"
         android:layout_alignParentLeft="true"
         android:layout_alignParentStart="true"
-        android:textSize="20sp"
+        android:layout_below="@id/arabicVerse"
+        android:background="#CCCCCC"
+        android:paddingBottom="6dp"
         android:paddingLeft="30dp"
         android:paddingRight="30dp"
         android:paddingTop="6dp"
-        android:paddingBottom="6dp" />
+        android:text="New Text"
+        android:textColor="#FFFFFF"
+        android:textSize="20sp" />
 
-    <ListView
+    <Spinner
+        android:id="@+id/translationSpinner"
         android:layout_width="match_parent"
-        android:layout_height="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/resultCount"
+        android:paddingLeft="12dp" />
+
+    <ListView
         android:id="@+id/listView"
-        android:paddingTop="0dp"
-        android:paddingBottom="10dp"
-        android:layout_alignParentBottom="true"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
This line is causing an issue with all searches that don't have enough results to fill the page. When you click a result to expand it, the expansion becomes confined to the original result size. Remove this line.